__init__.py

```
@@ -243,18 +244,24 @@ def match_common_query(self, phrase: str, lang: str) -> Tuple[str, float]:
         self.session_results[sess.session_id]["how_to"] = how_to
         return response, 0.7
 
-    def stop_session(self, sess: Session) -> bool:
+    def can_stop(self, message: Message) -> bool:
```

Ruff (0.11.9)

Undefined name `Message`

(F821)